torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -52,7 +51,8 @@ def empty(
     Returns:
         A :class:`ShardedTensor` object on each rank
     """
-    tensor_init_params = TensorInitParams(create_op=CreateOp.EMPTY, dtype=dtype, layout=layout,
+    tensor_init_params = TensorInitParams(create_op=CreateOp.EMPTY, fill_value=0,
```

Comment:
Do we need to specify `fill_value` here? We can omit it and it will just be `None` by default?

Reply:
Great point, done